client/src/templates/Challenges/redux/code-storage-epic.js

```diff
@@ -136,9 +137,18 @@ function loadCodeEpic(action$, state$) {
       const fileKeys = challengeFiles.map(x => x.fileKey);
       const invalidForLegacy = fileKeys.length > 1;
       const { title: legacyKey } = challenge;
-
       const codeFound = getCode(id);
-      if (codeFound && isFilesAllPoly(codeFound)) {
+
+      // see if this challenge is saved in database
```

Comment:
```suggestion
      // first check if the store (which is syncronized with the db) has saved
      // code
```
I didn't immediately get this, so an expanded comment might help.